### DISCUSSION TITLE
[A-L] (14 Luglio 2023) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#394] [Esami A-L]
Consideriamo la seguente relazione $\Large \prec$ definita su coppie di intervalli chiusi dell’insieme dei numeri interi relativi:

<div align="center">

$\Large [x, y] \prec [w, z]$ se e solo se $\Large [x, y] \subseteq [w, z]$ oppure $\Large y < z$.

</div>

Indicare se le seguenti affermazioni sono vere o false.

1. La relazione $\Large \prec$ è riflessiva. [1 punto]

2. La relazione $\Large \prec$ non è simmetrica. [2 punti]

3. La relazione $\Large \prec$ è transitiva. [2 punti]

**Comment by cek753** [comment]
V
V
V
